src/intl/en/page-contributing-translation-program-acknowledgements.json

```diff
@@ -0,0 +1,41 @@
+{
+  "page-contributing-translation-program-acknowledgements-acknowledgement-page-title": "Acknowledgement Page",
```

Comment:
"Acknowledgement Page" feels like a weird page title to me. Maybe something more "human", like "Acknowledging our contributors"?